src/components/ProfileSwitchModal/ProfileSwitchModal.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from "react";
+import { View, Text, Modal, Pressable, FlatList, Image } from 'react-native';
+import styles from './styles'
+
+import globalStyles from "../../globalStyles";
+import SvgIcon from "../../../assets/SvgIcon";
+
+function Item({
```

Comment:
Switch to const = () => {} format now that you no longer have linter issues!
Also think this might be better set as another component.